ui/packages/catalog-ui-search/src/main/webapp/js/widgets/cesium.line.js

```diff
@@ -53,7 +53,7 @@ class LineRenderView extends GeometryRenderView {
         json.lineWidth,
         model.get('lineUnits')
       ) || 1
-    if (!linePoints) {
+    if (!linePoints || !linePoints[0]) {
```

Comment:
:pencil2: 
```suggestion
    if (!linePoints || linePoints[0] !== undefined) {
```

Reply:
and same for the other cases in cesium.polygon, openlayers.line and openlayers.polygon